applicationconfig/client/README.md

```diff
@@ -0,0 +1,84 @@
+# App Configuration client library for Java
+Azure App Configuration allows developers to store, retrieve and manage access to application settings all in one place. It is easy to set up and simple to use from any application. It gives developers the ability to modify an application's behavior on demand without having to redeploy the application.
+
+Use the client library for Application Configuration to:
+- Adding a new configuration
+- Updating an existing configuration
+- Retrieving a configuration
+- Deleting a configuration
+
+[Source code]() | [Package (Maven)]() | [API reference documentation]() | [Product documentation](https://docs.microsoft.com/en-us/azure/azure-app-configuration/)
+
+## Getting started
+
+### Prerequisites
+
+- JDK 8 or later
+- Maven
+- Azure Service Principal
+- App Configuration resource
+
+### Adding the package to your product
+
+```xml
+<dependency>
+    <groupId>com.azure</groupId>
+    <artifactId>azure-app-configuration</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
+</dependency>
+```
+
```

Comment:
This document is missing https://github.com/Azure/azure-sdk/blob/master/docs/README-TEMPLATE.md#authenticate-the-client